src/modules/map/component.ts

```diff
@@ -122,7 +122,8 @@ export async function createMapComponent(components: {
         estates.resolve(addEstates(result.estates, {}))
         tokens.resolve(addTokens(result.parcels, result.estates, {}))
         ready = true
-        setTimeout(poll, refreshInterval)
+        await sleep(refreshInterval)
+        await poll()
```

Comment:
mm i would use `await` for `poll()` since it calls itself recursively, it would never resolve (it would create an endless chain of promises). I would rather move the `events.emit(MapEvents.READY, result)` from the line below up, above the `await sleep` and let the call to `poll` without awaits.. we could still catch it i guess, although the whole body of `poll` is already wrapped in a try/catch so shouldn't throw